src/components/viewCities.js

```diff
@@ -0,0 +1,108 @@
+import { Link } from 'react-router-dom';
+import { BsArrowRightCircle } from 'react-icons/bs';
+
+const citiesData = [
+  {
+    name: 'London',
+    lat: 44.34,
+    lon: 10.99,
+    country: 'GB',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/europe/gb/512.png',
+  },
+  {
+    name: 'Paris',
+    lat: 48.8589,
+    lon: 2.32,
+    country: 'FR',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/europe/fr/512.png',
+  },
+  {
+    name: 'Rome',
+    lat: 41.8933,
+    lon: 12.4829,
+    country: 'IT',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/europe/gr/512.png',
+  },
+  {
+    name: 'Spain',
+    lat: 40.4167,
+    lon: -3.7036,
+    country: 'ES',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/europe/es/512.png',
+  },
+  {
+    name: 'Berlin',
+    lat: 52.517,
+    lon: 13.3889,
+    country: 'DE',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/europe/de/512.png',
+  },
+  {
+    name: 'Tokyo',
+    lat: 35.6767,
+    lon: 139.764,
+    country: 'JP',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/asia/jp/512.png',
+  },
+  {
+    name: 'Beijing',
+    lat: 39.9062,
+    lon: 116.3913,
+    country: 'CN',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/asia/cn/512.png',
+  },
+  {
+    name: 'Singapore',
+    lat: 1.2899,
+    lon: 103.8519,
+    country: 'SG',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/asia/sg/512.png',
+  },
+  {
+    name: 'Mumbai',
+    lat: 19.0785,
+    lon: 72.8782,
+    country: 'IN',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/asia/in/512.png',
+  },
+  {
+    name: 'New York',
+    lat: 39.6853,
+    lon: -93.9269,
+    country: 'US',
+    img: 'https://raw.githubusercontent.com/djaiss/mapsicon/master/us/us/512.png',
+  },
+];
+
+const City = () => (
+  <section>
+    {citiesData.map((city) => (
+      <Link
+        key={city.country}
+        to={{
+          pathname: `/detail/${city.name}/${city.lat}/${city.lon}`,
+        }}
+      >
+        <div className="city">
+          <div className="map-arrow">
+            <img src={city.img} alt="city-map" />
+            <span className="right-arrow"><BsArrowRightCircle /></span>
+          </div>
+          <div className="city-info">
+            <h2>{city.name}</h2>
+            <p className="coordinate">
+              <span>Lat: </span>
+              <span>{city.lat}</span>
+            </p>
+            <p className="coordinate">
+              <span>Lon: </span>
+              <span>{city.lon}</span>
+            </p>
+          </div>
+        </div>
+      </Link>
+    ))}
+  </section>
```

Comment:
- [ Required] I think it would be better to stick to the project requirements as much as possible, it's a requirement that the items listed on the `list of items` should be from an `API`. Currently, the data displayed on the homepage has been hard coded in the `citiesData` array. I would like to propose that you use the value of the `details` key from the `redux store` through the use of the `useSelector` function. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-03-16 20-23-08](https://user-images.githubusercontent.com/16392046/225704667-9f25d762-6b5d-4ca9-8a82-a4d5e6c61628.png)

Reply:
Ok, I'll try.

Reply:
Hi @Felix45, Can you please elaborate that how can I get multiple cities' data even without passing different longitudes and altitudes? Because there it is required in API that we need to pass coordinates to get data.
![api](https://user-images.githubusercontent.com/99732138/225808902-1f0bb306-289d-4d0f-9226-057e18e3365c.png)

Reply:
Hi @MAleemH, 
I think it would be better to use two `API's` one that fetches data about `cities` e.g latitude & longitude, and then the one you are currently using to show the weather of a place. As per the requirements of the project, the data listed on the listing page needs to come from an API, currently, you are using an array.